seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/sink/file/ClickhouseFileSink.java

```diff
@@ -17,177 +17,33 @@
 
 package org.apache.seatunnel.connectors.seatunnel.clickhouse.sink.file;
```

Comment:
I am a little worried whether clickhousefile can still work properly because we lack test cases for this.

Reply:
@Hisoka-X  Do we need to add it in this PR